Makefile

```diff
@@ -17,12 +18,12 @@ all: format lint check_types test
 .PHONY: .install_common	
 .install_common:
 	@echo "Installing common Python dependencies"
-	uv sync --all-extras
+	uv sync
 
 .PHONY: install_common_ci
 install_common_ci:
 	@echo "[CI] Installing common Python dependencies"
-	uv sync --inexact --frozen --no-cache --quiet
+	uv sync --inexact --frozen --quiet
```

Comment:
The caching might blow up the disk space during the CI beyond our quota. Test seem to pass, so I'm fine with this change